### PR TITLE
pack as dotnet-tool

### PR DESCRIPTION
**Pull request by kMutagene**
This PR packages the arc commander as dotnet-tool that can be installed via `dotnet tool install`. To make that possible for users the nuget package has to be released though. This is one step in the release of this tool, self contained binaries will be added later. Additionally, i added fake tasks to test the packaged tool.

### CODE REVIEW

**Comment by HLWeil** [comment]
references #5

File: src/ArcCommander/ArcCommander.fsproj [hunk 1]
```diff
@@ -5,6 +5,20 @@
     <TargetFramework>netcoreapp3.1</TargetFramework>
 	<GenerateDocumentationFile>true</GenerateDocumentationFile>
   </PropertyGroup>
+  <PropertyGroup>
+    <PackAsTool>true</PackAsTool>
+    <ToolCommandName>ArcCommander</ToolCommandName>
+    <Authors>Heinrich Lukas Weil, Kevin Schneider, Timo Muehlhaus nfdi4plants OSS contributors</Authors>
```
Comment (HLWeil) [inline src/ArcCommander/ArcCommander.fsproj:11]: `,` missing after "Timo Mühlhaus"?

File: src/ArcCommander/ArcCommander.fsproj [hunk 2]
```diff
@@ -5,6 +5,20 @@
...
+    <Description>Tool to manage your ARCs</Description>
+    <Summary>Tool to manage your ARCs</Summary>
```
Comment (HLWeil) [inline src/ArcCommander/ArcCommander.fsproj:13]: ``Summary`  can be dropped

File: src/ArcCommander/ArcCommander.fsproj [hunk 3]
```diff
@@ -5,6 +5,20 @@
...
+    <PackageLicenseExpression>MIT</PackageLicenseExpression>
+    <PackageProjectUrl>https://github.com/nfdi4plants/arcCommander</PackageProjectUrl>
+    <PackageTags>rdm dotnet arc</PackageTags>
+    <RepositoryUrl>https://github.com/nfdi4plants/arcCommander</RepositoryUrl>
+    <RepositoryType>git</RepositoryType>
+    <FsDocsLicenseLink>https://github.com/nfdi4plants/arcCommander/blob/developer/LICENSE</FsDocsLicenseLink>
```
Comment (HLWeil) [inline src/ArcCommander/ArcCommander.fsproj:19]: I think `FsDocsLicenseLink` can be dropped as its deprecated

**Review by HLWeil — COMMENTED** [review]
Nice! 
I just commented some things in the package properties

Additionally, could you take a look at #37? Have you experience with implementing such a behaviour?

File: src/ArcCommander/ArcCommander.fsproj (hunk 3 above)
Comment (kMutagene) [reply src/ArcCommander/ArcCommander.fsproj:19]: https://fsprojects.github.io/FSharp.Formatting/styling.html#Customizing-via-Styling-Parameters

File: src/ArcCommander/ArcCommander.fsproj (hunk 1 above)
Comment (kMutagene) [reply src/ArcCommander/ArcCommander.fsproj]: Done

File: src/ArcCommander/ArcCommander.fsproj (hunk 2 above)
Comment (kMutagene) [reply src/ArcCommander/ArcCommander.fsproj]: Done

**Pull request merged**